.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "webextensions": true
+  },
+  "plugins": ["svelte3", "unicorn"],
```

Comment:
`eslint-plugin-unicorn` is not a direct dependency

Reply:
https://github.com/sveltejs/eslint-plugin-svelte3/issues/41#issuecomment-572503966

I commented the code now